### PR TITLE
Merge Vermillion updates to gh-pages

### PR DESCRIPTION
**Pull request by yandthj**

### CODE REVIEW

**Review by tdthatcher — DISMISSED** [review]
Looks very reasonable, especially as a starting point for a little bit of reorganization later.

**Review by tdthatcher — APPROVED** [review]

File: docs/Documentation/Systems/Vermillion/applications.md [hunk 1]
```diff
@@ -5,4 +5,69 @@ parent: Vermillion
 grand_parent: Systems
 ---
 
+The Vermilion HPC cluster marries traditional HPC deployments and modern cloud architectures.
 # Applications
+A list of packages available from the OpenHPC Community repo can be found at:
+
+    http://repos.openhpc.community/OpenHPC/2/EL_8/aarch64/
+
+Vermilion currently includes a subset of these packages in the base deployment. These include:
```
Comment (diskwarrior) [inline docs/Documentation/Systems/Vermillion/applications.md:14]: This list could very easily become out of date. Is there a better way to veiw/caputre available packages?

File: docs/Documentation/Systems/Vermillion/systemconfig.md [hunk 2]
```diff
@@ -6,3 +6,15 @@ grand_parent: Systems
 ---
 
 # System Configuration
+##
+
+---
+### Vermilion Partitions
```
Comment (diskwarrior) [inline docs/Documentation/Systems/Vermillion/systemconfig.md:12]: Partition chart could easily be created in system overview or elsewhere with more description.

**Review by diskwarrior — COMMENTED** [review]
Started before merge, however, late to the party. @yandthj might convert these to issues?

**Pull request merged**